src/modules/workspace/hooks/useIsFilteringInProgress.ts

```diff
@@ -1,24 +1,16 @@
-import { useTransactionsContext } from '@/modules/transactions/providers/TransactionsProvider';
-
 export type IUseIsFilteringInProgress = {
   isGifAnimationLoading: boolean;
+  isHomeFetching: boolean;
+  isTransactionsPageListFetching: boolean;
+  isVaultTransactionsFetching: boolean;
 };
 
 export const useIsFilteringInProgress = ({
   isGifAnimationLoading,
+  isHomeFetching,
+  isTransactionsPageListFetching,
+  isVaultTransactionsFetching,
 }: IUseIsFilteringInProgress) => {
-  const {
-    homeTransactions: {
-      request: { isFetching: isHomeFetching },
-    },
-    transactionsPageList: {
-      request: { isFetching: isTransactionsPageListFetching },
-    },
-    vaultTransactions: {
-      request: { isFetching: isVaultTransactionsFetching },
-    },
-  } = useTransactionsContext();
-
   const isFilteringInProgress =
     (isHomeFetching ||
```

Comment:
create a new const to OR logics here, and change `isFilteringInProgress`
`const isFilteringInProgress  = newConst && !isGifAnimationLoading;`

Reply:
Done